src/icons/general/IconBoxSquare.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+
+type Props = {
+    className?: string,
+    height?: number,
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconBoxSquare = ({ className = '', height = 72, title, width = 72 }: Props) => (
+    <AccessibleSVG
+        className={`box-square-icon ${className}`}
+        height={height}
+        title={title}
+        viewBox="0 0 72 72"
+        width={width}
+    >
+        <rect width="72" height="72" rx="8" ry="8" fill="#0061d5" />
+        <path
+            d="M37.39 28.57A10.26 10.26 0 0 0 28.33 34a10.31 10.31 0 0 0-15.23-3.39V22A2.05 2.05 0 0 0 9 22v17a10.27 10.27 0 0 0 19.33 4.62 10.25 10.25 0 1 0 9.06-15zM19.26 44.91a6.13 6.13 0 1 1 6.15-6.13 6.14 6.14 0 0 1-6.15 6.13zm18.13 0a6.13 6.13 0 1 1 6.15-6.13 6.14 6.14 0 0 1-6.15 6.13z"
+            fill="#fff"
```

Comment:
Color is hard coded? Should it be passed in like the other file?

Reply:
Hard coded because it's the box logo - so color shouldn't be variable. I can check that this box blue is set somewhere as a variable though

Reply:
Import from here https://github.com/box/box-ui-elements/blob/master/src/styles/variables.js

Reply:
Dont we need inverted box logo at times? Not sure how much of a use case is that
<img width="190" alt="Screen Shot 2019-04-15 at 11 42 23 AM" src="https://user-images.githubusercontent.com/1075325/56157121-952ad200-5f73-11e9-98e0-b7d477c1f632.png">

Reply:
I'll check with design on whether there's use cases for using other colors in the logo. If there's a need, I'll pass in variable

Reply:
From Zach: "I believe it always shows with Box Blue. Even though you can custom brand the UI, I think for branding purposes the logo should always be this way" 
Maybe we can set this logo as the box blue, and if there's a use case for custom branding the logo, someone can use the basic logo with css